plugin/wakatime.vim

```diff
@@ -572,6 +572,17 @@ EOF
             let cmd = cmd + ['--extra-heartbeats']
         endif
 
+        " Debugging category support
+        if has('nvim') || has('lua')
```

Comment:
Not super familiar with vimscript, maybe this should be `&&`

Reply:
Yep, I think `&&` so both expressions need to be true to satisfy the conditional. `||` means only one expression needs to be true.

Reply:
I think you only need to check for `has('lua')` because it doesn't actually matter if we're using NeoVim as long as the current environment supports Lua plugins.